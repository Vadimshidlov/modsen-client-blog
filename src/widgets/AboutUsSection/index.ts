export { AboutUsSection } from '@/widgets/AboutUsSection/ui/AboutUsSection';
