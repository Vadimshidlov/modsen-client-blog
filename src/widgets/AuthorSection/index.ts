export { AuthorSection } from '@/widgets/AuthorSection/ui/AuthorSection';
