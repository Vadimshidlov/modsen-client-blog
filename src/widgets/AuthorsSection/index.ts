export { AuthorsSection } from '@/widgets/AuthorsSection/ui/AuthorsSection';
