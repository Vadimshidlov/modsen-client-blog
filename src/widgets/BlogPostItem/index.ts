export { BlogPostItem } from '@/widgets/BlogPostItem/BlogPostItem';
