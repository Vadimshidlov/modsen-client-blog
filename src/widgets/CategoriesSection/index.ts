export { CategoriesSection } from '@/widgets/CategoriesSection/ui/CategoriesSection';
