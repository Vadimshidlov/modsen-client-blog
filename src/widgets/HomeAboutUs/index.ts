export { HomeAboutUs } from '@/widgets/HomeAboutUs/ui/HomeAboutUs';
