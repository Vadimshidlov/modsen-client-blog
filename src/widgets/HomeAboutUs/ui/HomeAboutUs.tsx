import React from 'react';
import Link from 'next/link';
import { Inter } from 'next/font/google';
import { useTranslations } from 'next-intl';
import container from '@/shared/styles/container.module.scss';
import styles from '@/widgets/HomeAboutUs/ui/HomeAboutUs.module.scss';

const inter = Inter({
  subsets: ['latin'],
  weight: '600',
  variable: '--font-inter',
});

export function HomeAboutUs() {
  const t = useTranslations('HomeAboutUs');

  return (
    <section className={`${styles.about__section} ${container.container}`}>
      <div className={styles.colorBarContainer}>
        <div className={styles.firstColorBar} />
        <div className={styles.secondColorBar} />
      </div>
      <div className={styles.contentContainer}>
        <article className={styles.aboutBlock}>
          <h4 className={`${styles.aboutSubTitle} ${inter.variable}`}>{t('subTitle')}</h4>
          <h2 className={styles.aboutTitle}>{t('title')}</h2>
          <span className={styles.aboutText}>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
            incididunt ut labore et dolore magna aliqua.
          </span>
          <Link href="/about">
            <span className={styles.aboutButton}>{t('buttonText')} &gt;</span>
          </Link>
        </article>
        <article className={styles.missionBlock}>
          <h4 className={`${styles.missionSubTitle} ${inter.variable}`}>{t('missionSubTitle')}</h4>
          <h2 className={styles.missionTitle}>{t('missionTitle')}</h2>
          <span className={styles.missionText}>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
            exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
          </span>
        </article>
      </div>
    </section>
  );
}
