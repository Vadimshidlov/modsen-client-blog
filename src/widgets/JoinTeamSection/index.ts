export { JoinTeamSection } from '@/widgets/JoinTeamSection/ui/JoinTeamSection';
