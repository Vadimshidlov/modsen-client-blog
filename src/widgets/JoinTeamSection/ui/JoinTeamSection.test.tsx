import React from 'react';
import { render } from '@testing-library/react';
import { HomeAboutUs } from '@/widgets/HomeAboutUs';

jest.mock('next-intl', () => ({
  useTranslations: jest.fn().mockReturnValue((key: string) => `Mocked ${key}`),
}));

describe('HomeAboutUs tests', () => {
  test('should matches snapshot', () => {
    const { container } = render(<HomeAboutUs />);
    expect(container).toMatchSnapshot();
  });
});
