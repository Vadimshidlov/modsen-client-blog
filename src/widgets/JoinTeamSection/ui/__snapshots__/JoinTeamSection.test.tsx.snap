// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAboutUs tests should matches snapshot 1`] = `
<div>
  <section
    class="about__section container"
  >
    <div
      class="colorBarContainer"
    >
      <div
        class="firstColorBar"
      />
      <div
        class="secondColorBar"
      />
    </div>
    <div
      class="contentContainer"
    >
      <article
        class="aboutBlock"
      >
        <h4
          class="aboutSubTitle variable"
        >
          Mocked subTitle
        </h4>
        <h2
          class="aboutTitle"
        >
          Mocked title
        </h2>
        <span
          class="aboutText"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        </span>
        <a
          href="/about"
        >
          <span
            class="aboutButton"
          >
            Mocked buttonText
             &gt;
          </span>
        </a>
      </article>
      <article
        class="missionBlock"
      >
        <h4
          class="missionSubTitle variable"
        >
          Mocked missionSubTitle
        </h4>
        <h2
          class="missionTitle"
        >
          Mocked missionTitle
        </h2>
        <span
          class="missionText"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
        </span>
      </article>
    </div>
  </section>
</div>
`;
